src/features/Movies/Movies.api.ts

```diff
@@ -0,0 +1,37 @@
+import { LanguageType } from "store/language/types";
+import { getCountryCode } from "utils";
+import { THE_MOVIE_DB_BASE_URL } from "constants/api";
+import { SelectedFiltersType } from "./types";
+
+const { REACT_APP_THE_MOVIE_DB_KEY } = process.env;
+
+export const getGenres = async (activeLanguage: LanguageType) => {
+  const countryCode = getCountryCode(activeLanguage);
+
+  const response = await fetch(
+    `${THE_MOVIE_DB_BASE_URL}/genre/movie/list?api_key=${REACT_APP_THE_MOVIE_DB_KEY}&language=${countryCode}`
+  );
+  const { genres } = await response.json();
+
+  return genres;
+};
+
+export const getMoviesByFilters = async (
+  activeLanguage: LanguageType,
+  selectedFilters: SelectedFiltersType,
+  page?: number
+) => {
+  const countryCode = getCountryCode(activeLanguage);
+  const { genreId, startDate, endDate } = selectedFilters!;
+
+  const response = await fetch(
+    `${THE_MOVIE_DB_BASE_URL}/discover/movie?${
```

Comment:
It's very hard to read this string, I don't even try to do this. Use `query-string`, `qs` or similar libraries here